docs/source/03_quickstart.rst

```diff
@@ -439,6 +439,19 @@ In case you need to customize pipeline run context, modifying configuration file
 - ``--pipeline`` allows to select a pipeline to run (by default, the ``__default__`` pipeline is started),
 - ``--params`` takes a JSON string with parameters override (JSONed version of ``conf/*/parameters.yml``, not the Kedro's ``params:`` syntax),
 - ``--env-var KEY=VALUE`` sets the OS environment variable injected to the steps during runtime (can be used multiple times).
+- ``--load-versions`` specifies a particular dataset version (timestamp) for loading (similar behavior as Kedro)
+- ``--on-job-scheduled  path.to.module:my_function`` specifies a callback function to be called on the azureml pipeline job start (example below)
+
+.. code:: python
```

Comment:
This does not display in the docs after rendering (see the screen) 👇🏻

You're probably missing a blank line after `.. code:: python`, see the previous usages in the same file above.

<img width="732" alt="image" src="https://github.com/getindata/kedro-azureml/assets/6958772/27647b04-c201-42e7-9d42-972104a81ea4">